<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Markov chains  by  Temuujin Yanjinlham</string>
  </property>
  <widget class="QGroupBox" name="groupBoxInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>391</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Input transition matrix</string>
   </property>
   <widget class="QTableWidget" name="tableWidgetMatrix">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>451</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>3613</width>
      <height>301</height>
     </size>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>10</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>15</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>10</number>
    </attribute>
   </widget>
   <widget class="QLabel" name="labelStatesCount">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>30</y>
      <width>143</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Number of states:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxStatesCount">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>53</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxResult">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>471</width>
     <height>331</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Stationary distribution calculation </string>
   </property>
   <widget class="QPushButton" name="pushButtonCalc">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ð¡alculate</string>
    </property>
   </widget>
   <widget class="QComboBox" name="chooseMethod">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Repeated matrix multiplication</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Finding left Eigen vectors</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelMethod">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Method:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="resultTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>451</width>
      <height>241</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>210</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>210111</width>
      <height>80000</height>
     </size>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>225</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Stationary distribution</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Stabilization time</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>501</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vlTimeGraph"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>430</y>
     <width>501</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vlStateGraph"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
